src/client/components/hover-multi-bubble/hover-multi-bubble.tsx

```diff
@@ -26,53 +26,46 @@ const LEFT_OFFSET = 22;
 
 export interface ColorEntry {
   color: string;
-  segmentLabel: string;
-  measureLabel: string;
+  name: string;
+  value: string;
+  delta?: JSX.Element;
```

Comment:
Should we keep delta as JSX.Element or as plain types as other properties?

Reply:
It should be complex type because it has complex structure inside. I will try to narrow it to accept only outputs from Delta component. Other option is to define it as `DeltaAttributes` and apply component in `renderColorSwabs`

Reply:
Ok. Limiting possible JSX.Element inputs would be difficult and could obscure intent. I stand by decision that we should accept complex node from outside. Maybe someone wants to display delta in bubble with different icons or something. Decision should be on call site.